packages/web3/test/e2e/e2e_utils.ts

```diff
@@ -50,6 +50,12 @@ export const getE2ETestAccountAddress = (): string => {
 	throw new Error('Unable to get test account address');
 };
 
+export const getE2ETestContractAddress = (): string => {
+	return getSystemTestBackend() === 'sepolia'
+		? secrets['SEPOLIA']['DEPLOYED_TEST_CONTRACT_ADDRESS']
+		: secrets['MAINNET']['DEPLOYED_TEST_CONTRACT_ADDRESS'];
+};
+
```

Comment:
```suggestion
export const getE2ETestContractAddress = (): string => 
	process.env[getSystemTestBackend().toUpperCase() as 'SEPOLIA' | 'MAINNET']
	.DEPLOYED_TEST_CONTRACT_ADDRESS;
```

Matches the other util functions in this file